playbooks/roles/jumpbox/files/mysql_db_backup.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
```

Comment:
- [ ] You may want to converge both files since they are similar and handle diffs via params